source/Api.cpp

```diff
@@ -49,22 +46,10 @@ namespace Aws
             aws_event_stream_library_init(allocator);
             aws_sdkutils_library_init(allocator);
 
-            cJSON_Hooks hooks;
-            hooks.malloc_fn = s_cJSONAlloc;
-            hooks.free_fn = s_cJSONFree;
-            cJSON_InitHooks(&hooks);
-        }
-
-        ApiHandle::ApiHandle(Allocator *allocator) noexcept
-            : m_logger(), m_shutdownBehavior(ApiHandleShutdownBehavior::Blocking)
-        {
-            s_initApi(allocator);
+            JsonObject::OnLibraryInit();
```

Comment:
not something to fix in this pr, but we should think how to make this better. 
hooks in json are global and we are breaking abstraction boundary when code upsteam from us depends on cjson as well. ex. cpp sdk overrides the hooks we set

Reply:
maybe something to make the symbols unique...
find/replace in the source code
or fancy interning trickery